interface/patient_file/report/custom_report.php

```diff
@@ -308,7 +308,44 @@ function postToGet($arin) {
     //
     if (stristr($key,"include_")) {
 
-        if ($val == "demographics") {
+        if($val == "recurring_days"){
+
+            /// label/header for recurring days
+            echo "<hr />";
+            echo "<div class='text' id='appointments'>\n";
+            print "<h1>".xl('Recurrent Appointments').":</h1>";
+
+            //fetch the data of the recurring days
+            $recurrences = fetchRecurrences($pid);
+
+            //print the recurring days to screen
+            if($recurrences[0] == false){ //if there are no recurrent appointments:
+                echo "<div class='text' >";
+                echo "<span>" . xlt('None') . "</span>";
+                echo "</div>";
+                echo "<br>";
+            }
+            else {
+                foreach ($recurrences as $row) {
+                    //checks if there are recurrences and if they are current (git didn't end yet)
+                    if ($row == false || !recurrence_is_current($row['pc_endDate']))
+                        continue;
```

Comment:
Logic:
1. The first if is skipped, thus 'None' is not shown.
2. If all items in $recurrences are not current (ie. get skipped by ($row == false || !recurrence_is_current($row['pc_endDate'])))

Then you will have just a blank screen with no entries and no 'None'